widgets/post-carousel/post-carousel.php

```diff
@@ -25,8 +25,17 @@ function sow_carousel_get_next_posts_page() {
 		)
 	);
 
+	$instance_hash = $_GET['instance_hash'];
+	$template_vars = array();
+	if ( ! empty( $instance_hash ) ) {
+		$widget = new SiteOrigin_Widget_PostCarousel_Widget();
```

Comment:
Yeah, I think you mentioned that previously and I fixed it up [here](https://github.com/siteorigin/so-widgets-bundle/pull/412/commits/75c8c7b1f26624991c2f1fa477290a8f3d538a9e) :smile: